app/controllers/shipit/stacks_controller.rb

```diff
@@ -4,6 +4,7 @@ class StacksController < ShipitController
 
     def new
       @stack = Stack.new
+      render 'new_bs', layout: 'shipit_bs' if Shipit.feature_bootstrap? || params[:layout] == 'bootstrap'
```

Comment:
I think we could factorize this with [`prepend_view_path`](http://api.rubyonrails.org/classes/ActionView/ViewPaths/ClassMethods.html#method-i-prepend_view_path).

It would allow to make new versions of the view without having to add these conditionals in every actions. I'll take a stab at this tomorrow.